install.sh

```diff
@@ -57,33 +96,65 @@ chroot_install() {
         || exit 1
     chmod +x ./alpine-chroot-install || exit 1
     ./alpine-chroot-install -d /alpine-v310 -b v3.10
-    /alpine-v310/enter-chroot install.sh
+    /alpine-v310/enter-chroot ./install.sh
+}
+
+check_os() {
+    if [ -f "/etc/alpine-release" ]; then
+        alpine_install
+        return 0
+    fi
+
+    if [ -x "$(command -v apt-get)" ]; then
```

Comment:
https://github.com/lncm/noma/pull/48/files#diff-3fbb47e318cd8802bd325e7da9aaabe8R31 checks for the same thing, but in a different way.  Perhaps it's better to decide on one way?